proxlb

```diff
@@ -279,6 +285,66 @@ def api_connect(proxmox_api_host, proxmox_api_user, proxmox_api_pass, proxmox_ap
     return api_object
 
 
+def __api_connect_get_host(proxmox_api_host):
+    """ Validate if a list of API hosts got provided and pre-validate the hosts. """
+    info_prefix  = 'Info: [api-connect-get-host]:'
+    proxmox_port = 8006
+
+    if ',' in proxmox_api_host:
+        logging.info(f'{info_prefix} Multiple hosts for API connection are given. Testing hosts for further usage.')
+        proxmox_api_host =  proxmox_api_host.split(',')
+
+        # Validate all given hosts and check for responsive on Proxmox web port.
+        for host in proxmox_api_host:
+            logging.info(f'{info_prefix} Testing host {host} on port tcp/{proxmox_port}.')
+            reachable = __api_connect_test_ipv4_host(host, proxmox_port)
+            if reachable:
+                return host
+    else:
+        logging.info(f'{info_prefix} Using host {host} on port tcp/{proxmox_port}.')
```

Comment:
Replace `host` with `proxmox_api_host`. Host is only defined in the loop.

(only note to me to do it in the evening)